src/ansys/fluent/core/solver/flobject.py

```diff
@@ -189,7 +189,7 @@ def set_flproxy(self, flproxy):
         """Set flproxy object."""
         self._setattr("_flproxy", flproxy)
 
-    def set_file_transfer_service(self, file_transfer_service):
+    def _set_file_transfer_service(self, file_transfer_service):
         """Set file_transfer_service."""
```

Comment:
```suggestion
        """Set the file transfer service."""
```
It is best practice to use plain English in descriptions whenever possible.